app/views/statuses/_group.html.erb

```diff
@@ -0,0 +1,13 @@
+<div class="status-group">
+  <%= render 'statuses/status', status: group %>
+
+  <div class="status-items">
+    <% group.statuses.each do |status| %>
```

Comment:
:+1:. It was not clear to me that `StatusGroup` were being used in the regular `_status` partial. But just saw the render call on the top of the file after your explanation.